monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -20,23 +21,27 @@ class CreateZuoraSubscription(servicesProvider: ServiceProvider = ServiceProvide
 
   def this() = this(ServiceProvider)
 
-  override protected def servicesHandler(state: CreateZuoraSubscriptionState, context: Context, services: Services): Future[SendThankYouEmailState] =
+  override protected def servicesHandler(
+    state: CreateZuoraSubscriptionState,
+    requestInfo: RequestInfo,
+    context: Context,
+    services: Services
+  ): FutureHandlerResult =
     services.zuoraService.getRecurringSubscription(state.user.id, state.contribution.billingPeriod).flatMap {
-      case Some(sub) => skipSubscribe(state, sub)
-      case None => subscribe(state, services)
+      case Some(sub) => skipSubscribe(state, requestInfo, sub)
+      case None => subscribe(state, requestInfo, services)
     }
 
-  def skipSubscribe(state: CreateZuoraSubscriptionState, subscription: SubscriptionResponse): Future[SendThankYouEmailState] = {
-    logger.info(s"Skipping subscribe for user because they are already an active contributor " +
```

Comment:
Have we lost this logging, or does it happen in appendMessage?

Reply:
I left it out, now that the message will appear in the console it didn't seem that useful.

Reply:
I'd prefer to keep it in the logs as well, if possible. It's useful to keep track of how often this is happening as it could be symptomatic of a wider problem.

I was actually thinking we might even want to add a metric or something around this. Although it's valuable as a safety check, theoretically we could break this check and stop taking money altogether, and we'd struggle to notice because everything would stay green!